### PR TITLE
feat: added new get involved section to homepage

### PR DESCRIPTION
**Pull request by atepem**
## Proposed changes

Draft for now, as it may require a few minor tweaks but would like a review on the current state. Removed previous get-involved section and implemented new layout according to the Figma design. 

## Screenshots/Recordings
Before:
image

After:
image

### CODE REVIEW

**Review by oaksprout — APPROVED** [review]

**Comment by Tanya-atatakai** [comment]
@atepem can we ensure that links and titles are in one line with each other? as in figma 
image

image

File: components/HomepageSection/GetInvolved.jsx [hunk 1]
```diff
@@ -4,72 +4,95 @@ import Link from 'next/link';
...
-    ctaText: 'Get OLAS',
-    href: '/olas-token',
-  },
-  {
-    id: 3,
     imageSrc: '/images/homepage/olas-contribute.png',
-    title: 'Grow awareness about Olas, earn points',
+    title: 'Grow awareness about Olas',
+    description:
+    'Prmote Olas on X and earn points while doing so.',
```
Comment (Tanya-atatakai) [inline components/HomepageSection/GetInvolved.jsx:15]: there's a small typo in promote. you can easily copy the text from figma by right clicking on it 
image

Comment (atepem) [reply components/HomepageSection/GetInvolved.jsx]: Oh my bad 😓 Good catch

**Review by Tanya-atatakai — APPROVED** [review]
nice!

**Review by oaksprout — APPROVED** [review]
Looks good!

**Review by Tanya-atatakai — APPROVED** [review]

**Review by oaksprout — APPROVED** [review]

**Pull request merged**